src/compat/db_client_type.hpp

```diff
@@ -43,8 +43,8 @@ enum db_client_type
   DB_CLIENT_TYPE_RO_BROKER_REPLICA_ONLY = 12,
   DB_CLIENT_TYPE_SO_BROKER_REPLICA_ONLY = 13,
   DB_CLIENT_TYPE_ADMIN_CSQL_WOS = 14,	/* admin csql that can write on standby */
-  DB_CLIENT_TYPE_SKIP_VACUUM_CSQL = 15,	
-  DB_CLIENT_TYPE_SKIP_VACUUM_ADMIN_CSQL = 16,	
+  DB_CLIENT_TYPE_SKIP_VACUUM_CSQL = 15,
+  DB_CLIENT_TYPE_SKIP_VACUUM_ADMIN_CSQL = 16,
```

Comment:
unrelated change

Reply:
Thanks, it was because of the git hook for c++ syntax. I reverted it on 9831a76